maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -330,6 +330,14 @@
     private String forkMode;
 
     /**
+     * Directory for temporary test files. Be removed after tests execute. Value is relative path from reportsDirectory parent
```

Comment:
Rewrite the text to this one:
"Relative path to <i>project.build.directory</i> containing internal Surefire temporary files. It is removed after the test set has completed."